src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+
+    public AuthenticationController(AuthenticationService authenticationService) {
+        this.authenticationService = authenticationService;
+    }
+
+    @PostMapping("/register")
+    public void register(@RequestBody UserRequestDto userRequest) {
+        authenticationService.register(userRequest.getEmail(), userRequest.getPassword());
+    }
```

Comment:
I think you should return created user in response dto, in order to provide generated id